infra/gcp/terraform/README.md

```diff
@@ -47,6 +47,7 @@ containing manifests that are deployed to the cluster
 
 ## Deploying
 
+- Open a PR and Atlantis will apply and deploy your Terraform changes.
```

Comment:
(maybe we should be clear that this will happen _after merge_ just to be explicit?)